mesh/mesh.cpp

```diff
@@ -5053,6 +5053,42 @@ std::vector<int> Mesh::CreatePeriodicVertexMapping(
    }
    return v2v;
 }
+void Mesh::RefineNURBSFromFile(std::string ref_file)
```

Comment:
There should be 1 empty line before the function.

Reply:
Fixed